pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -698,7 +711,6 @@ def cmd_instance_references(context, instancename, options):
                 instancepath,
                 ResultClass=options['resultclass'],
                 Role=options['role'],
-                IncludeQualifiers=options['includequalifiers'],
                 IncludeClassOrigin=options['includeclassorigin'],
                 PropertyList=resolve_propertylist(options['propertylist']))
```

Comment:
IncludeQualifiers is missing as an argument.

Reply:
Fixed in release late last night